prebuild/action.yml

```diff
@@ -2,10 +2,14 @@ runs:
   using: composite
   steps:
     - uses: actions/download-artifact@v3
+    - name: Set unsafe for alpine
```

Comment:
I would remove the parts using npm and just use yarn to install `node-gyp` since we use `yarn` for everything outside the target repository.

Reply:
`${PACKAGE_MANAGER}` should be used when interacting with the repository that invoked the action, that's why `$PACKAGE_MANAGER install  --ignore-scripts` should be kept (we need to respect the choice of the repository on how to install its dependencies).
But to install node-gyp globally, we can use yarn whatever the value of `${PACKAGE_MANAGER}`.

Reply:
My bad, I didn't notice that the host directory was being mounted in the container. I reverted the changes in that part.

Reply:
yeah, I find this code hard to reason about